src/test/ui/consts/const-eval/validate_uninhabited_zsts.32bit.stderr

```diff
@@ -10,14 +10,11 @@ LL |     unsafe { std::mem::transmute(()) }
 LL | const FOO: [Empty; 3] = [foo(); 3];
    |                          ----- inside `FOO` at $DIR/validate_uninhabited_zsts.rs:13:26
 
-error[E0080]: it is undefined behavior to use this value
-  --> $DIR/validate_uninhabited_zsts.rs:16:1
+error[E0080]: evaluation of constant value failed
+  --> $DIR/validate_uninhabited_zsts.rs:16:35
    |
 LL | const BAR: [Empty; 3] = [unsafe { std::mem::transmute(()) }; 3];
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ type validation failed at [0]: encountered a value of uninhabited type Empty
-   |
```

Comment:
I did the same [in a Miri test](https://github.com/rust-lang/miri/pull/2078/files#diff-f422314ad7e7c2105673b9fe090e4798ca3d64b938602bbb6bf14c4736a2b107) so that type definition can probably just be copied.